src/pickup.cpp

```diff
@@ -251,6 +251,116 @@ static bool select_autopickup_items( std::vector<item> &here, std::vector<bool>
     return bFoundSomething;
 }
 
+
+// parameter definition - append_fail_msg - any other message you want to append to the error message displayed
```

Comment:
To amplify what @kevingranade said: in particular, the [Doxygen documentation we update automatically](http://dev.narc.ro/cataclysm/doxygen/) understands this style.